data-serving/scripts/convert_data.py

```diff
@@ -0,0 +1,55 @@
+# Lint as: python3
+'''Script to convert CSV line-list data into json compliant with the MongoDB schema.'''
+
+import argparse
+import csv
+import logging
+import json
+
+parser = argparse.ArgumentParser(
```

Comment:
Purely nit/IMO -- I'd stuff this in main() (or a helper function called there). I think it's better not to have "naked" (no class/function) logic.

Reply:
oops idk why i did that -- fixed!